J2WL0G/StudentRegistrySystem/DAO/IStudentDAO.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using StudentRegistrySystem.Model;
+
+namespace StudentRegistrySystem.DAO
+{
+    internal interface IStudentDAO
+    {
+        public void StudentPerformanceInSubject(int id, string subject);
+        public List<Student> GetStudents();
+        public void DisplayStudents();
+        public void DisplayHelps();
+        public bool AddStudent(Student student);
+        public bool RemoveStudent(int id);
+        public bool AddGrade(int id, int grade, string subjectName);
+        public bool AddSubject(int id, string subjectName);
+        public void BestAverageStudentsGradeInSubject(string subject);
```

Comment:
Szerintem ez a DAO Több dolgot is csinál, mint amit kéne neki. Pl Betöltés és mentés lehetne külön interfész, implementációja lehetne ugyan az az osztály akár